release-tools/prow.sh

```diff
@@ -86,7 +86,7 @@ configvar CSI_PROW_BUILD_PLATFORMS "linux amd64 amd64; linux ppc64le ppc64le -pp
 # which is disabled with GOFLAGS=-mod=vendor).
 configvar GOFLAGS_VENDOR "$( [ -d vendor ] && echo '-mod=vendor' )" "Go flags for using the vendor directory"
 
-configvar CSI_PROW_GO_VERSION_BUILD "1.22.5" "Go version for building the component" # depends on component's source code
+configvar CSI_PROW_GO_VERSION_BUILD "1.23.1" "Go version for building the component" # depends on component's source code
```

Comment:
Changes under release-tools need to be submitted to https://github.com/kubernetes-csi/csi-release-tools and get merged there first.  Then we can import those changes to all the sidecars.

Reply:
Yes, PR to csi-release-tools have been merged, and this PR is the importing.

Reply:
@huww98 Better to add related csi-release-tools PR in this

Reply:
https://github.com/kubernetes-csi/csi-release-tools/pull/261 . But also includes changes from previous PRs. See commit message for detail.

Reply:
Please copy the commit message from https://github.com/kubernetes-csi/external-snapshotter/pull/1208/commits/9f167db60c4f89bbfc6f96eb6761b9eb6d1afc8a to the PR description.

Reply:
Updated